# Frequency in which to generate velocity commands to the robot platform
frequency: 20

# transform frames
robot_frame: ropod/base_link
odometry_frame: ropod/odom 

# Maximum velocity command the operator will send to the robot (in m/s)
max_velocity: 1.0

# If set to true, the selected best direction is published as a GridCells-Message
publish_route: true

# The way is assumed to be free when there is no obstacle within this range (in meter)
max_free_space: 2.0
safety_decay: 0.99

# Take desired direction only if its safe to move there
desired_safety_threshold: 0.25

# The importance weights of the 3 action values
safety_weight: 1
escape_weight: 0.8
conformance_weight: 0.5
continue_weight: 0.8
